new/src/components/ScenarioOverview/ScenarioOverview.tsx

```diff
@@ -42,32 +44,93 @@ interface Headers {
-                    <Grid item>
-                        <ScenarioActionButtons targets={props.targets} />
-                    </Grid>
-                </Grid>
-            </ListItem>
-            <Divider />
-            <ListItem>
-                <ListItemText primary={StatisticBreadcrumbs(props)} />
-            </ListItem>
-            <ListItem>
-                <canvas id={"symbol-canvas"} width={"50"} height={"2"} />
-            </ListItem>
-        </List>
+        <div style={{ display: 'flex'}}>
+            <Content>
+                <List>
+                    <ListItem>
+                        <Grid container direction={"row"} justifyContent="flex-end" alignItems="flex-start">
+                            <Grid item xs={12} sm={8}>
+                                <ScenarioTitles headers={props.headers} />
+                            </Grid>
+                            <Grid item sx={{ flexGrow: 1 }} />
+                            <Grid item>{createReportCircle(props)}</Grid>
+                            <Grid item>
+                                <ScenarioActionButtons targets={props.targets} />
+                            </Grid>
+                        </Grid>
+                    </ListItem>
+                    <Divider />
+                    <ListItem>
+                        <ListItemText primary={StatisticBreadcrumbs(props)} />
+                    </ListItem>
+                    <ListItem>
+                        <canvas id={"symbol-canvas"} width={"50"} height={"2"} />
+                    </ListItem>
+                </List>
+            </Content>
+        </div>
     );
 }
 
+export function MenuBar() {
+    return(
+        <StyledDrawer variant="permanent">
+            <List>
+                <ListItem sx={{ paddingTop: 0.1, paddingBottom: 0.1 }}>
+                    <ListItemText primary={<Typography variant="h6">SUMMARY</Typography>} />
+                </ListItem>
+                <List>
+                    <ListItem sx={{ paddingTop: 0.1, paddingBottom: 0.1 }}>
+                        <ListItemText primary={
+                            <Link href="http://localhost:3000" underline="none" sx={{ color: 'inherit' }}>
+                                All Scenarios
+                            </Link>}
+                        />
+                    </ListItem>
+                    <ListItem sx={{ paddingTop: 0.1, paddingBottom: 0.1 }}>
+                        <ListItemText primary={
+                            <Link href="http://localhost:3000" underline="none" sx={{ color: 'inherit' }}>
+                                Failed Scenarios
+                            </Link>}
+                        />
+                    </ListItem>
+                    <ListItem sx={{ paddingTop: 0.1, paddingBottom: 0.1 }}>
+                        <ListItemText primary={
+                            <Link href="http://localhost:3000" underline="none" sx={{ color: 'inherit' }}>
```

Comment:
We clearly need to start adding routing...

Reply:
not part of this PR obv.

Reply:
Yeah.